netlify.toml

```diff
@@ -1192,4 +1192,27 @@
 # Added: 2022-02-21
 [[redirects]]
     from = "/blog/hipaa-compliant-analytics"
-    to = "/blog/best-hipaa-compliant-analytics-tools"
\ No newline at end of file
+    to = "/blog/best-hipaa-compliant-analytics-tools"
+
+# Added: 2022-03-03
+[[redirects]]
+    from = "/handbook/engineering/databases/async-migrations"
+    to = "/handbook/engineering/async-migrations"
+
+
+# Added: 2022-03-03
+[[redirects]]
+    from = "/handbook/engineering/databases/event-ingestion"
+    to = "/handbook/engineering/event-ingestion"
+
+
+# Added: 2022-03-03
+[[redirects]]
+    from = "/handbook/engineering/databases/query-performance-optimization"
+    to = "/handbook/engineering/query-performance-optimization"
+
+
+# Added: 2022-03-03
+[[redirects]]
+    from = "/handbook/engineering/databases/schema-changes"
+    to = "/handbook/engineering/schema-changes"
```

Comment:
Is this relevant to this PR?

Reply:
👍 This makes sense overall, though maybe not everything in here will require an RFC. Most importantly I think we should just keep the roadmap in the back of our heads when planning.

Reply:
Makes sense I won't overdo it and just get this list up in the handbook